packages/block/src/helpers.ts

```diff
@@ -152,16 +126,15 @@ export const getDataGasPrice = (header: BlockHeader) => {
 }
 
 /**
- * Returns the total fee for data gas spent on `numBlobs` in the current/pending block
- * @param numBlobs
- * @param parent parent header of the current/pending block
- * @returns the total data gas fee for a transaction assuming it contains `numBlobs`
+ * Returns the total fee for data gas spent in the current/pending block
+ * @param header of the current/pending block
+ * @param numBlobs number of blobs in the transaction
```

Comment:
Isn't this the `numBlobs` in the "block" and not the tx?

Reply:
it can be used for any (so will update), but in the EIP second arg is the `tx`

Reply:
Ah, that makes sense, sorry.  I was thinking this helper had been added to `blockheader`.